benchmarks/benchmark_visualize.py

```diff
@@ -0,0 +1,193 @@
+import argparse
+import matplotlib.pyplot as plt
+import string
+import re
+
+marks = {}
+marks["bs_mark"] = r"bs:"
+marks["iter_mark"] = r"iter:"
+marks["input_tokens_length_mark"] = r"input_tokens_length:"
+marks["prompts_num_mark"] = r"num_prompts:"
+marks["total_time_mark"] = r"Total time:"
+marks["prompt_length_mark"] = r"Prompt Length (Min/Med/Max):"
+marks["request_throughput_mark"] = r"Request Throughput (QPS):"
+marks["input_token_throughput_mark"] = r"Input Token Throughput:"
+marks["output_token_throughput_mark"] = r"output Token Throughput:"
+marks["latency_per_req_mark"] = r"Average latency per Request:"
+marks["latency_per_token_mark"] = r"Average latency per Token:"
+marks["latency_first_token_mark"] = r"Average latency for First Tokens:"
+marks["latency_next_token_mark"] = r"Average latency for Next Tokens:"
+
+
+def extract_metric(file_name, mark_name):
+    with open(file_name, "r") as file:
+        logs = file.read()
+    extract_value = re.findall(marks[mark_name] + r"\s+([\d.]+)", logs)
```

Comment:
The problem of parsing from pure text output is it's not very flexible and error-prone. 
It's better to output json/csv format as result. Then data analytics libs such as pandas can be used.